specification/sbi_atee.adoc

```diff
@@ -583,12 +505,12 @@ The possible error codes returned in `sbiret.error` are shown below.
 === Function: TEE Host Finalize TVM (FID #6)
 [source, C]
 ------
-struct sbiret sbi_tee_host_finalize_tvm(unsigned long tvm_guest_id);
+struct sbiret sbi_tee_host_finalize_tvm(unsigned long tvm_guest_id,
+                                        unsigned long entry_sepc,
+                                        unsigned long entry_arg);
 ------
 Transitions the TVM specified by `tvm_guest_id` from the `TVM_INITIALIZING` state to a `TVM_RUNNABLE`
-state. The host must finalize TVM shared-memory execution parameters like the entry point (`ENTRY_PC`)
-and boot argument (`ENTRY_ARG`) on the boot VCPU configured by `sbi_tee_host_create_tvm()` before making
-this call.
+state. Also, sets the entry point (`ENTRY_PC`) using `entry_sepc` and boot argument (`ENTRY_ARG`) using `entry_arg` for the boot VCPU.
```

Comment:
These configuration parameters are included in the measurement of the TVM.

Reply:
It would be good to clarify what exactly the entry_sepc & entry_arg should contain